context-free.md

```diff
@@ -1,22 +1,27 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+_Anyone who wants to create online generated art._
 
 ## What is easy to do in this language? Why is it easy?
-
+_It is easy to create shapes because you can just type in what you want and it will appear. 
+Also, it is easy to generate variations of the same shape with different parameters._
 
 ## What is hard to do in this language? Why is it hard?
-
+_I found it hard to put the shapes I created exaclty where I wanted. Having to toy with the 
```

Comment:
Yeah, I don't really think about it when using other drawing programs, but I can choose exactly where I want things to go by clicking on the canvas unlike here where you need to know the coordinates which can be confusing to figure out.